### DISCUSSION TITLE
rook-ceph-osd pods are not getting created, while the monitoring pods are created

### DISCUSSION THREAD

**Discussion by richaiter08** [rook/rook#9982] [General]
I am currently configuring the rook-ceph cluster and pods and services for storage to replicate fraud detection use-case with open data hub. 

But on setting the cluster on persistent volume chain yaml, the rook-ceph monitoring pods get created and OSD pods are not getting created. So I am unable to proceed further.

I am trying the same in OCP 4.7 cluster in opentlc

Please find snippet of the pods.

oc get pods -n rook-ceph
NAME                                                        READY   STATUS     RESTARTS   AGE
csi-cephfsplugin-55hqs                                      3/3     Running    0          7d4h
csi-cephfsplugin-provisioner-5c65b94c8d-krdx2               6/6     Running    0          7d4h
csi-cephfsplugin-provisioner-5c65b94c8d-tgs4j               6/6     Running    0          7d4h
csi-cephfsplugin-tvgwx                                      3/3     Running    0          7d4h
csi-rbdplugin-fgv5k                                         3/3     Running    0          7d4h
csi-rbdplugin-provisioner-569c75558-wz42j                   6/6     Running    0          7d4h
csi-rbdplugin-provisioner-569c75558-xjr48                   6/6     Running    0          7d4h
csi-rbdplugin-v5nd5                                         3/3     Running    0          7d4h
rook-ceph-crashcollector-ip-10-0-155-14-9588c8869-jr2j6     0/1     Init:0/2   0          57m
rook-ceph-crashcollector-ip-10-0-178-171-65cfcc6678-lp8p7   0/1     Init:0/2   0          57m
rook-ceph-mon-a-canary-6db88875b7-rpplm                     1/1     Running    0          2m
rook-ceph-mon-b-canary-5c8547f89-vgcnv                      1/1     Running    0          119s
rook-ceph-mon-c-canary-58c77798c-7hlws                      0/1     Pending    0          119s
rook-ceph-operator-59cbfb7c7c-xzsfz                         1/1     Running    0          7d4h
rook-discover-4c4c7                                         1/1     Running    0          7d4h
rook-discover-qhpbz                                         1/1     Running    0          7d4h

**Comment by travisn** [comment]
@richaiter08 It appears the mons are not forming quorum, or that the operator is waiting for the third mon which is stuck in Pending. What does `kubectl describe` show for the pending mon?
